code/modules/items/specials/ItemSpecials.dm

```diff
@@ -100,6 +100,19 @@
 
 	return ((dx*dx) + (dy*dy))
 
+/// Finds every mob that is currently moving away from a turf, but has not reached the end of their movement.
+/proc/atoms_in_combat_range(var/turf/target)
+	var/list/atom/atoms = list()
+	for(var/atom/A in target)
+		atoms += A
+	for(var/atom/A in range(1,target))
+		if(ismob(A))
```

Comment:
Filter for mobs in the loop statement instead of looping over atoms and then filtering to mobs